tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -37,7 +71,11 @@ public function set_up() {
 		parent::set_up();
 
 		$this->context                = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
-		$this->reader_revenue_manager = new Reader_Revenue_Manager( $this->context );
+		$this->options                = new Options( $this->context );
+		$this->user                   = $this->factory()->user->create_and_get( array( 'role' => 'administrator' ) );
+		$this->user_options           = new User_Options( $this->context, $this->user->ID );
```

Comment:
As these instances are not used outside the constructor, it isn't necessary to define class properties for them. Simple local variables should be sufficient.

```suggestion
		$options                      = new Options( $this->context );
		$user                         = $this->factory()->user->create_and_get( array( 'role' => 'administrator' ) );
		$user_options                 = new User_Options( $this->context, $this->user->ID );
```